### PR TITLE
[Translation] Review Serbian translations

### LINKED ISSUE #53776: Missing translations for Serbian (sr_Cyrl)
Hello,

There are some translation work needed for Serbian (sr_Cyrl) and we are looking for a **native** speaker to help us out. 

Here is a [short example](https://symfony-translations.nyholm.tech/#pr) of what you need to do. There are 4 rules: 

1. You must be a Serbian (sr_Cyrl) native speaker
2. You must look at the existing translations and follow the same "style" or "tone"
3. You must make your PR to branch 5.4
4. You must use the correct indentation (number of spaces)

These are the files that should be updated: 
- [src/Symfony/Component/Security/Core/Resources/translations/security.sr_Cyrl.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Security/Core/Resources/translations/security.sr_Cyrl.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.sr_Cyrl.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.sr_Cyrl.xlf)

<details>
<summary>Show related strings</summary>

#### src/Symfony/Component/Security/Core/Resources/translations/security.sr_Cyrl.xlf

| Id | English | Translation | Status |
| -- | -- | -- | -- |
| 20 | Too many failed login attempts, please try again in %minutes% minutes. | Превише неуспешних покушаја пријављивања, покушајте поново за %minutes% минут.|Превише неуспешних покушаја пријављивања, покушајте поново за %minutes% минута.|Превише неуспешних покушаја пријављивања, покушајте поново за %minutes% минута. | Needs review |

#### src/Symfony/Component/Validator/Resources/translations/validators.sr_Cyrl.xlf

| Id | English | Translation | Status |
| -- | -- | -- | -- |
| 37 | This is not a valid IP address. | Ова вредност није валидна IP адреса. | Needs review |
| 51 | No temporary folder was configured in php.ini. | Привремени директоријум није конфигурисан у php.ini, или конфигурисани директоријум не постоји. | Needs review |
| 59 | This is not a valid International Bank Account Number (IBAN). | Ова вредност није валидан Међународни број банкарског рачуна (IBAN). | Needs review |
| 81 | This is not a valid Business Identifier Code (BIC). | Ова вредност није валидан Код за идентификацију бизниса (BIC). | Needs review |
| 83 | This is not a valid UUID. | Ова вредност није валидан UUID. | Needs review |
| 113 | This URL is missing a top-level domain. | Овом URL недостаје домен највишег нивоа. | Needs review |

</details>

> [!NOTE]
> If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### LINKED ISSUE #53777: Missing translations for Serbian (sr_Latn)
Hello,

There are some translation work needed for Serbian (sr_Latn) and we are looking for a **native** speaker to help us out. 

Here is a [short example](https://symfony-translations.nyholm.tech/#pr) of what you need to do. There are 4 rules: 

1. You must be a Serbian (sr_Latn) native speaker
2. You must look at the existing translations and follow the same "style" or "tone"
3. You must make your PR to branch 5.4
4. You must use the correct indentation (number of spaces)

These are the files that should be updated: 
- [src/Symfony/Component/Security/Core/Resources/translations/security.sr_Latn.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Security/Core/Resources/translations/security.sr_Latn.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf)

<details>
<summary>Show related strings</summary>

#### src/Symfony/Component/Security/Core/Resources/translations/security.sr_Latn.xlf

| Id | English | Translation | Status |
| -- | -- | -- | -- |
| 20 | Too many failed login attempts, please try again in %minutes% minutes. | Previše neuspešnih pokušaja prijavljivanja, pokušajte ponovo za %minutes% minut.|Previše neuspešnih pokušaja prijavljivanja, pokušajte ponovo za %minutes% minuta.|Previše neuspešnih pokušaja prijavljivanja, pokušajte ponovo za %minutes% minuta. | Needs review |

#### src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf

| Id | English | Translation | Status |
| -- | -- | -- | -- |
| 37 | This is not a valid IP address. | Ova vrednost nije validna IP adresa. | Needs review |
| 51 | No temporary folder was configured in php.ini. | Privremeni direktorijum nije konfigurisan u php.ini, ili konfigurisani direktorijum ne postoji. | Needs review |
| 59 | This is not a valid International Bank Account Number (IBAN). | Ova vrednost nije validan Međunarodni broj bankovnog računa (IBAN). | Needs review |
| 81 | This is not a valid Business Identifier Code (BIC). | Ova vrednost nije validan Kod za identifikaciju biznisa (BIC). | Needs review |
| 83 | This is not a valid UUID. | Ova vrednost nije validan UUID. | Needs review |
| 112 | This value is not a valid MAC address. | Ova vrednost nije validna MAC adresa. | Needs review |
| 113 | This URL is missing a top-level domain. | Ovom URL nedostaje domen najvišeg nivoa. | Needs review |

</details>

> [!NOTE]
> If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by danilovict2**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | Fix #53776 Fix #53777
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Cyrl.xlf [hunk 1]
```diff
@@ -440,31 +440,31 @@
...
             </trans-unit>
             <trans-unit id="117">
                 <source>This value is not a valid week.</source>
-                <target state="needs-translation">This value is not a valid week.</target>
+                <target>Ова вредност није валидна недеља.</target>
             </trans-unit>
             <trans-unit id="118">
                 <source>This value should not be before week "{{ min }}".</source>
-                <target state="needs-translation">This value should not be before week "{{ min }}".</target>
+                <target>Ова вредност не би требала да буде пре недеље „{{ min }}“.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Cyrl.xlf]:
```suggestion
                <target>Ова вредност не би требала да буде пре недеље "{{ min }}".</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Cyrl.xlf [hunk 2]
```diff
@@ -312,15 +312,15 @@
             </trans-unit>
             <trans-unit id="81" resname="This is not a valid Business Identifier Code (BIC).">
                 <source>This value is not a valid Business Identifier Code (BIC).</source>
-                <target state="needs-review-translation">Ова вредност није валидан Код за идентификацију бизниса (BIC).</target>
+                <target>Ова вредност није валидна Код за идентификацију бизниса (BIS).</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Cyrl.xlf]: for confirmation: BIS and not BIC?

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 3]
```diff
@@ -312,15 +312,15 @@
             </trans-unit>
             <trans-unit id="81" resname="This is not a valid Business Identifier Code (BIC).">
                 <source>This value is not a valid Business Identifier Code (BIC).</source>
-                <target state="needs-review-translation">Ova vrednost nije validan Kod za identifikaciju biznisa (BIC).</target>
+                <target>Ova vrednost nije validan Kod za identifikaciju biznisa (BIC).</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf:315]: BIC or BIS?

**Comment by fabpot** [comment]
Thank you @danilovict2.

**Pull request merged**